src/locationTable.lua

```diff
@@ -58,11 +59,17 @@ function LocationTable:__tostring()
 	return string
 end
 
+local function computeAbsCoords(x, y, angle)
+	local r = vector.magnitude(x, y)
+	local t = vector.angle(x, y)
+	return vector.components(r, t + angle)
+end
```

Comment:
This function belongs in vector. I would rename it to something like rotate vector, so little more universal.

Reply:
That makes sense. I'll do that.